pkg/watch/dockerignore_test.go

```diff
@@ -0,0 +1,92 @@
+package watch
```

Comment:
You need to add the license header to this new file

Reply:
thank you, pushed fix